TASK_1:
  EDGE: [1, 1]
  TYPE: "go_to_goal_predicate_2d"
  CENTER: [8.0, 7.0]
  EPSILON: 2.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [20, 50]
  INVOLVED_AGENTS: [1]
